sdk/textanalytics/out/azure/ai/textanalytics/_models.pyi

```diff
@@ -119,7 +118,6 @@ class LinkedEntity(DictMixin):
     id: Any = ...
     url: Any = ...
     data_source: Any = ...
-    is_error: bool = ...
```

Comment:
I'm guessing this was removed because this is intended to be an outgoing object type?

Reply:
Only the result and error objects should have the `is_error` attribute. I accidentally added it to this model in preview 1.